tests/main/sbuild/task.yaml

```diff
@@ -0,0 +1,21 @@
+summary: Ensure snapd builds correctly in sbuild
+
+# takes a while
+priority: 500
+
+systems: [debian-sid-*]
+
+execute: |
+    echo "Ensure we have sbuild"
+    apt install -y sbuild eatmydata
+
+    echo "Create a sid sbuild env"
+    eatmydata sbuild-createchroot --include=eatmydata,ccache,gnupg sid /srv/chroot/sid-amd64-sbuild http://deb.debian.org/debian
```

Comment:
This is one of the places we could integrate the proxy information we set in spread.yaml. This would allow for efficient execution of this test in qemu.

Reply:
Good point, the http{,s}_proxy env should be set already - what is needed in addition to this?

Reply:
Hmm, I think if you have a generic proxy that works for anything this is sufficient. In practice I never do, I always have apt-specific proxy and I need to tell sbuild to use it explicitly by using something like:
```
sbuild-createchroot --include=eatmydata,ccache,gnupg sid /srv/chroot/sid-amd64-sbuild ${apt_proxy}/debian${apt_proxy_port}
 ```